src/VideoRemise/VideoRemise/VideoGridManager.cs

```diff
@@ -159,11 +179,14 @@ private void OnLightStatus(object sender, Trigger.LightEventArgs args)
             }
             if (args.LightsOn != Lights.None)
             {
+                /*
```

Comment:
Eventually this needs to come "back to life" to handle the replay loop definition properly.

Reply:
What do you mean by replay loop definition?

Reply:
A replay loop -- an action, if you prefer -- is defined by a start offset, an end offset, and a file name. The start offset is easy, you subtract an offset (from config) from the first event that marks the action. The end is a bit more complicated because you need to wait some amount of time after the last event that's part of the action (for instance, if there are two separate lights in a foil bout, you want to wait some amount of time after the second light).